module/module.go

```diff
@@ -757,25 +724,50 @@ func (m *Module) AddStream(ctx context.Context, req *streampb.AddStreamRequest)
 		return nil, errors.Wrap(err, "error creating a new TrackLocalStaticRTP")
 	}
 
-	subID, err := vcss.SubscribeRTP(ctx, rtpBufferSize, func(pkts []*rtp.Packet) error {
+	sub, err := vcss.SubscribeRTP(ctx, rtpBufferSize, func(pkts []*rtp.Packet) {
```

Comment:
I can't comment off-diff. IIUC, `Module.AddStream` only succeeds when the underlying stream (aka camera) supports rtp passthrough. If that's correct, can we add that limitation to the `Module.AddStream` docstring?

Reply:
done